scripts/run_locally.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+echo "==> $(date +%H:%M:%S) ==> Setting up environment variables from sample file..."
+TEMP=temp.env \
+  && cp ../.env.sample $TEMP \
+  || exit
+
+echo "==> $(date +%H:%M:%S) ==> Using http://polygon-rpc.com as default RPC node"
+sed -i '' 's|http://url.to.node|https://polygon-rpc.com|g' $TEMP \
```

Comment:
I wouldn't do it here, I don't feel it very intuitive to tell the user to edit this file to get the node they want, and in the case we want it maybe set `http://polygon-rpc.com` inside a variable, like `NODE_URL=http://polygon-rpc.com` so you can reuse it also in the `echo`

Reply:
Yes, I agree, I thought in the file like something the user won't touch, but maybe it's a bit confusing, in the case they actually want to change the RPC url, to modify this file. What do you think it's a better option here? 

1. Setting a `NODE_URL` variable inside this file.
2. Just including the default RPC url in the `.env.sample` file.
3. Not setting a default at all, so the script will fail if the users don't set the RPC url themselves.
4. Do you see another option?

🙂

Reply:
I think users should configure `.env` as stated in the docs with the node they need and the rest of the configuration should be going on from there

Reply:
I've removed `.env` manipulation from the script. Now it just starts Docker containers and manages superuser creation 🙂